src/modules/admin/components/Permissions/ColonyPermissionsAddDialog.tsx

```diff
@@ -27,10 +28,11 @@ import {
   getAllRootAccounts,
   getUserRolesForDomain,
 } from '../../../transformers';
-import { availableRoles } from './constants';
-import PermissionForm from './PermissionForm';
-
-import styles from './ColonyPermissionsDialog.css';
+import {
+  PermissionManagementForm,
+  availableRoles,
+  styles,
+} from '../../../dashboard/components/PermissionManagementDialog';
```

Comment:
Please import from the alias:
```suggestion
} from '~dashboard/PermissionManagementDialog';
```